examples/fdc3-trade-simulator/js-order-book/src/index.html

```diff
@@ -7,7 +7,6 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
The CDN link for material icons should be also removed like the above, to avoid CI errors resulting from time outs.
If the icons are used in the frontend they can be installed as an npm package (material-icons), and added as an import to this file: https://github.com/lilla28/ComposeUI/blob/fix/fdc3-joinChannel/examples/fdc3-trade-simulator/js-trader-app/src/styles.scss, like in the Process Explorer frontend: https://github.com/morganstanley/ComposeUI/blob/main/prototypes/process-explorer/oss-frontend/src/styles.scss#L3